Sacramento Bee.js

```diff
@@ -0,0 +1,247 @@
+{
+	"translatorID": "82b534ea-abf0-4527-9d5d-48cd98a89ba5",
+	"label": "Sacramento Bee",
+	"creator": "czar",
+	"target": "^https?://(www\\.)?sacbee\\.com",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2017-07-08 22:56:15"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2017 czar
+	http://en.wikipedia.org/wiki/User_talk:Czar
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+// attr()/text()
+function attr(doc,selector,attr,index){if(index>0){var elem=doc.querySelectorAll(selector).item(index);return elem?elem.getAttribute(attr):null}var elem=doc.querySelector(selector);return elem?elem.getAttribute(attr):null}function text(doc,selector,index){if(index>0){var elem=doc.querySelectorAll(selector).item(index);return elem?elem.textContent:null}var elem=doc.querySelector(selector);return elem?elem.textContent:null}
+
+function detectWeb(doc, url) {
+	if (url.match(/article\d+/)) {
+		return "newspaperArticle";
+	} else if (url.match(/\/(news|sports|entertainment)\//)) {
+		return "multiple";
+	} else if (url.indexOf("/search/?q=") != -1) {
+		return "multiple";
+	} else return null;
+}
+
+function scrape(doc, url) {
+	var item = new Zotero.Item("newspaperArticle");
+	item.websiteTitle = "The Sacramento Bee";
+	item.language = "en-US";
+	item.ISSN = "0890-5738";
+	item.url = url;
+	item.title = attr(doc,'[property="og:title"]','content');
+	item.date = text(doc,'.published-date');
+	item.abstractNote = text(doc,'#content-body- p');
+	item.tags = attr(doc,'meta[name="keywords"]','content').split(", ");
```

Comment:
`attr()` will return null if the selector isn't found, which would produce an error, so it'd be safer to assign it to a `keywords` variable and then do `if (keywords) { item.tags = keywords.split(", "); }`.